### ISSUE TITLE
Error running LightDarkLPDM

### ISSUE THREAD

**Issue by ChandlerKilpatrick1** [ebalaban/LightDarkLPDM#5]
Hey @ebalaban , here is the error I get when I run execute.jl (I am trying to get this to run in Julia 1.0+ so I can use it for reference when making VDPTagLPDM): 

```Julia
Expected State:
"4.66\n"
Root node lb=-173.26, ub=-21.02, uba*=0.00
Before: lBound = -173.2640773207, uBound = -21.0207373328
ERROR: LoadError: type LDBounds1d has no field rng
Stacktrace:
 [1] getproperty(::Main.execute.LDBounds1d{Float64,Float64,Float64}, ::Symbol) at ./Base.jl:20
 [2] search(::LPDM.LPDMSolver{Float64,Float64,Float64,Main.execute.LDBounds1d{Float64,Float64,Float64},LPDM.RNGVector}, ::LightDarkLPDM.LightDark1DLpdm) at /Users/chandlerkilpatrick/.julia/dev/LPDM/src/solver.jl:81
 [3] action(::LPDM.LPDMPolicy{Float64,Float64,Float64,Main.execute.LDBounds1d{Float64,Float64,Float64}}, ::LPDM.LPDMBelief{Float64,Float64,Float64}) at /Users/chandlerkilpatrick/.julia/dev/LPDM/src/LPDM.jl:177
 [4] macro expansion at /Users/chandlerkilpatrick/.julia/dev/LightDarkLPDM/src/execute.jl:269 [inlined]
 [5] macro expansion at ./util.jl:288 [inlined]
 [6] #run_scenario#2(::Int64, ::Int64, ::Array{Int64,1}, ::typeof(Main.execute.run_scenario), ::Float64, ::Main.execute.LPDMTest, ::Type, ::Type, ::Type, ::LPDM.LPDMConfig) at /Users/chandlerkilpatrick/.julia/dev/LightDarkLPDM/src/execute.jl:259
 [7] (::Main.execute.var"#kw##run_scenario")(::NamedTuple{(:dims, :n_sims, :vis),Tuple{Int64,Int64,Array{Int64,1}}}, ::typeof(Main.execute.run_scenario), ::Float64, ::Main.execute.LPDMTest, ::Type, ::Type, ::Type, ::LPDM.LPDMConfig) at ./none:0
 [8] #batch_execute#1(::Int64, ::typeof(Main.execute.batch_execute)) at /Users/chandlerkilpatrick/.julia/dev/LightDarkLPDM/src/execute.jl:174
 [9] (::Main.execute.var"#kw##batch_execute")(::NamedTuple{(:dims,),Tuple{Int64}}, ::typeof(Main.execute.batch_execute)) at ./none:0
 [10] top-level scope at /Users/chandlerkilpatrick/.julia/dev/LightDarkLPDM/src/test_lightdark.jl:8
in expression starting at /Users/chandlerkilpatrick/.julia/dev/LightDarkLPDM/src/test_lightdark.jl:8

```

I am guessing this has to do with an LPDM change that was made but I am not sure what to update. Do I just need to get the latest master branch of LPDM? Thanks.

**Comment by ebalaban** [comment]
Will fix later today.

**Comment by ebalaban** [comment]
Fixed with 732d2161eb996109207e4d5a01cad44f54d0079d.
